README.md

```diff
@@ -24,3 +24,20 @@ Specific rules are configured via [.markdownlint.yaml](./.markdownlint.yaml).
 Additionally, there is a npm script `lint-doc`, that will lint all the markdown files inside [docs](./docs) and `lint-kits`, that will lint all the markdown files inside [docs-kits](./docs-kits).
 This script is also run as a pre-commit hook, set up via [husky](https://www.npmjs.com/package/husky).
 You can also run the linting step manually by running `npm run lint-doc` or `npm run lint-kits`.
+
+## Licence Checks
+
+This repository enforces licence compatibility checks on all Pull Requests. Follow these steps if
```

Comment:
To give the users more content, I would add a link with more information, like TRG 7.04:
https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04#checking-libraries-using-the-eclipse-dash-license-tool

Reply:
added